sorc/ncep_post.fd/SURFCE.f

```diff
@@ -2706,12 +2707,12 @@ SUBROUTINE SURFCE
            cfld=cfld+1
            fld_info(cfld)%ifld=IAVBLFLD(IGET(508))
            fld_info(cfld)%lvl=LVLSXML(1,IGET(508))
-           fld_info(cfld)%tinvstat=1
-           if (IFHR > 0) then
+           if(ITSRFC>0) then
              fld_info(cfld)%ntrange=1
            else
              fld_info(cfld)%ntrange=0
            endif
+           fld_info(cfld)%tinvstat=IFHR-ID(18)
```

Comment:
@EricJames-NOAA I don't see definition of ID(18) in the block of index 508. Can you add it?